src/main/java/seedu/address/AlfredException.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address;
+
+public abstract class AlfredException extends Exception {
+
+
+    public AlfredException(String message) {
+        super(message);
+    }
+
+    public AlfredException(String message, Throwable cause) {
```

Comment:
Just curious, why have cause?

Reply:
It’s an abstract class with all the different constructors for other exception to extend from. Other exceptions need a throwable cause to indicate source of error